src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -0,0 +1,692 @@
+	}
+
+	private JScrollPane getScrollPaneInventory() {
+		if (scrollPaneInventory == null) {
+			scrollPaneInventory = new JScrollPane();
+			scrollPaneInventory.setViewportView(getJTableInventory());
+		}
+		return scrollPaneInventory;
+	}
+
+	private JTable getJTableInventory() {
+		if (jTableInventory == null) {
+			jTableInventory = new JTable();
+			jTableInventory.setFillsViewportHeight(true);
+			jTableInventory.setModel(new InventoryBrowsingModel(0, PAGE_SIZE));
+			jTableInventory.setAutoCreateColumnsFromModel(false);
+			for (int i = 0; i < columwidth.length; i++) {
+				jTableInventory.getColumnModel().getColumn(i).setMinWidth(columwidth[i]);
+				if (columnCentered[i]) {
+					jTableInventory.getColumnModel().getColumn(i).setCellRenderer(new ColorCenterTableCellRenderer());
+				} else {
+					jTableInventory.getColumnModel().getColumn(i).setCellRenderer(new ColorTableCellRenderer());
+				}
+			}
+			jTableInventory.getSelectionModel().addListSelectionListener(new ListSelectionListener() {
+
+				@Override
+				public void valueChanged(ListSelectionEvent e) {
+					if (e.getValueIsAdjusting()) {
+						int[] selectedRows = jTableInventory.getSelectedRows();
+						if (selectedRows.length == 1) {
+							int selectedRow = jTableInventory.getSelectedRow();
+							MedicalInventory inventory = inventoryList.get(selectedRow);
+							if (inventory.getStatus().equals(InventoryStatus.canceled.toString()) ||
+								inventory.getStatus().equals(InventoryStatus.done.toString())) {
+								jButtonEdit.setEnabled(false);
+								jButtonDelete.setEnabled(false);
+							} else {
+								jButtonEdit.setEnabled(true);
+								jButtonDelete.setEnabled(true);
+
+							}
+							jButtonView.setEnabled(true);
+							jButtonView.setEnabled(true);
+							jButtonDelete.setEnabled(true);
+						} else {
+							jButtonEdit.setEnabled(false);
+							jButtonView.setEnabled(false);
+							jButtonDelete.setEnabled(false);
+						}
+					}
+
+				}
+			});
+		}
+		return jTableInventory;
+	}
+
+	class InventoryBrowsingModel extends DefaultTableModel {
+
+		private static final long serialVersionUID = 1L;
+
+		public InventoryBrowsingModel(int page, int pageSize) {
+			inventoryList = new ArrayList<>();
+			String state = statusComboBox.getSelectedIndex() > 0 ? statusComboBox.getSelectedItem().toString().toLowerCase() : null;
+			String type = InventoryType.main.toString();
+			try {
+				Page<MedicalInventory> medInventorypage = medicalInventoryManager.getMedicalInventoryByParamsPageable(dateFrom, dateTo, state, type, page,
+					pageSize);
+				inventoryList = medInventorypage.getContent();
+			} catch (OHServiceException e) {
+				OHServiceExceptionUtil.showMessages(e);
+			}
+		}
+
+		public Class< ? > getColumnClass(int c) {
+			if (c == 0) {
+				return String.class;
+			} else if (c == 1) {
+				return String.class;
+			} else if (c == 2) {
+				return String.class;
+			} else if (c == 3) {
+				return String.class;
+			}
```

Comment:
if the value of `c` is greater than or equal to 0 and less then 4 it always returns `String.class`.  Multiple `ifs` are not needed.